Documentation/ApiOverview/CachingFramework/Developer/_MyClass.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Vendor\MyExtension;
+
+use TYPO3\CMS\Core\Cache\Frontend\FrontendInterface;
+
+final class MyClass
+{
+    public function __construct(
+        private readonly FrontendInterface $cache
+    ) {
+    }
+
+    //...
+
+    private function getCachedValue(string $cacheIdentifier, array $data, array $tags, int|null $lifetime)
```

Comment:
Hmm, I don't like the example. Had to looked at it more than once and don't get the concrete use case: If I already have the data, why storing and retrieving it from the cache?

The original example calculated "expensive" values. This is missing here and should be adjusted.

Reply:
Like this?